src/app/common/use-wallet-type.ts

```diff
@@ -0,0 +1,36 @@
+import { useCurrentKeyDetails } from '@app/store/keys/key.selectors';
+import { useMemo } from 'react';
+import { noop } from 'react-query/types/core/utils';
```

Comment:
Getting a build error here locally, is this import correct?

Reply:
This should be the locally defined noop, not the one from react query